pkg/core/consensus/queue.go

```diff
@@ -68,13 +86,15 @@ func (eq *Queue) Clear(round uint64) {
 	defer eq.lock.Unlock()
 
 	newEntries := make(map[uint64]map[uint8][]message.Message)
+	eq.items = 0
 
 	for r := range eq.entries {
 		if r > round {
 			newEntries[r] = make(map[uint8][]message.Message)
 
 			for s, m := range eq.entries[r] {
 				newEntries[r][s] = m
+				eq.items++
```

Comment:
Maybe this should be `eq.items += len(m)` ?

Reply:
Good catch!